modules/security/pages/password-policy.adoc

```diff
@@ -1,32 +1,121 @@
 = Password Policy
```

Comment:
Great work @zhihuapengtg . Please review my comments and make changes soon. Thanks!

Reply:
@zhihuapengtg  - This PR should be attached to appropriate DOC JIRA ticket. Can you please do it?

Reply:
Yes. https://graphsql.atlassian.net/browse/DOC-2103.
Has mentioned, thanks!